icons/rainmeter.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Rainmeter icon</title><path d="M12 0l-.7 1.088S9.49 3.91 7.677 7.205c-.908 1.647-1.818 3.413-2.51 5.025-.692 1.613-1.191 3.029-1.191 4.225C3.975 20.647 7.609 24 12 24s8.025-3.353 8.025-7.545c0-1.196-.499-2.612-1.191-4.225-.692-1.612-1.602-3.378-2.51-5.025-1.814-3.295-3.627-6.117-3.627-6.117L12 0zm0 3.168c.507.804 1.435 2.239 2.87 4.842.89 1.618 1.78 3.348 2.437 4.879.657 1.53 1.056 2.91 1.056 3.566 0 3.226-2.807 5.879-6.363 5.879-3.555 0-6.363-2.653-6.363-5.879 0-.657.4-2.035 1.056-3.566.657-1.531 1.547-3.261 2.438-4.88C10.564 5.408 11.493 3.973 12 3.169zM9.262 14.609c-1.533-.077-2.15 1.16-2.15 1.16a3.033 3.033 0 00-.06.55c0 2.547 2.216 4.613 4.948 4.613s4.947-2.066 4.947-4.614a3.05 3.05 0 00-.06-.548s-1.946 2.215-4.938 0c-1.104-.818-1.99-1.125-2.687-1.16z"/></svg>
```

Comment:
I'm spotting some slight differences between your PR (red) and source - built from the .AI file (black).
![image](https://user-images.githubusercontent.com/1830867/103438877-ea1ee800-4c2f-11eb-9490-a473c9e96966.png)

Having generated the path from in Illustrator, I've been able to get it working with a precision of 3 - would you mind altering the path to this, and we can merge this PR?
```svg
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Rainmeter icon</title><path d="M12.7 1.088L12 0l-.7 1.088c-.751 1.168-7.342 11.525-7.342 15.366C3.958 20.615 7.565 24 12 24s8.042-3.385 8.042-7.546c0-3.84-6.591-14.197-7.342-15.366zM12 22.335c-3.516 0-6.377-2.638-6.377-5.881C5.623 13.908 9.732 6.756 12 3.1c2.268 3.656 6.377 10.809 6.377 13.355 0 3.242-2.861 5.88-6.377 5.88zm4.957-6.017c0 2.548-2.22 4.615-4.957 4.615-2.737 0-4.957-2.067-4.957-4.615 0-.163.021-.347.058-.549 0 0 1.306-2.616 4.847 0 2.999 2.215 4.95 0 4.95 0 .038.202.059.386.059.549z"/></svg>
```